site/content/protocols/mediator-coordination/2.0/readme.md

```diff
@@ -22,6 +22,12 @@ There are two roles in this protocol:
 - `mediator`: The agent that will be receiving `forward` messages on behalf of the _recipient_.
 - `recipient`: The agent for whom the `forward` message payload is intended.
 
+## Requirements
+
+The `return_route` extension must be supported by both agents (`recipient` and `mediator`).
+The common use of this protocol is for the reply messages from the `mediator` to be synchronous (utilizing the same connection chennal for the reply).
+In order to have this synchronous behavior the `recipient` should specify `return_route` header to `all`.
+
```

Comment:
Suggest the following for this section (and matching in the other protocols):

The [`return_route` extension](https://github.com/decentralized-identity/didcomm-messaging/blob/main/extensions/return_route/main.md) must be supported by both agents (`recipient` and `mediator`).
The common use of this protocol is for the reply messages from the `mediator` to be synchronous, utilizing the same connection channel for the reply. In order to have this synchronous behavior the `recipient` should specify `return_route` header to `all`. This header must be set each time the communication channel is established: once per established websocket, and every message for an HTTP POST.